doc: align landing pages with standard pattern (#1337)
canonical/microcloud: doc/how-to/index.md

(howto)=
# How-to guides

These MicroCloud how-to guides cover key operations and processes.

> 

Reviewer: ```suggestion
```

> ## Get started

Reviewer: ```suggestion
## Initial setup
```
I prefer this because "Get started" is a common phrase used by many other docs sets across Canonical (and beyond) to refer to their tutorial. (I don't like that LXD uses "Get started" in their how-to index either, and this could be updated during the merge.)

Examples:
- https://canonical-microceph.readthedocs-hosted.com/latest/tutorial/get-started/
- https://canonical-openstack.readthedocs-hosted.com/en/latest/tutorial/get-started-with-openstack/
- https://documentation.ubuntu.com/pebble/tutorial/getting-started/
- https://snapcraft.io/docs/tutorials/get-started/

MicroCloud's own tutorial also uses "Get started".

environment, and learn how to initialize MicroCloud through interactive or
automated configuration processes. You can also set up access to the MicroCloud
UI.

```{toctree}
:maxdepth: 1

Install MicroCloud </how-to/install>
Initialize MicroCloud </how-to/initialize>
Access the UI </how-to/ui>
```

## Configure services

You can configure storage with MicroCeph and networking with MicroOVN during the
initialization process, or you can add a service later.

```{toctree}
:maxdepth: 1

Configure Ceph networking </how-to/ceph_networking>
Configure OVN underlay </how-to/ovn_underlay>
Add a service </how-to/add_service>
```

## Manage clusters and cluster members

As your needs change, follow these steps to manage your clusters and cluster
members and keep your deployment up to date.

```{toctree}
:maxdepth: 1

Add a cluster member </how-to/member_add>
Remove a cluster member </how-to/member_remove>
Shut down a cluster member </how-to/member_shutdown>
Manage multiple clusters </how-to/cluster_manager>
Recover MicroCloud </how-to/recover>
Update and upgrade </how-to/update_upgrade>
Manage the snaps </how-to/snaps>
```

## Automated deployment with Terraform

> Follow this guide to learn how to use Terraform to automate the deployment of a

Reviewer: ```suggestion
Follow this guide to use Terraform to automate the deployment of a
```

MicroCloud test environment. As in our {ref}`introductory tutorial

> <tutorial-multi>`, this test environment uses Virtual Machines (VMs) as

Reviewer: ```suggestion
<tutorial-multi>`, this test environment uses virtual machines (VMs) as
```

> MicroCloud test environment. As in our {ref}`introductory tutorial
> <tutorial-multi>`, this test environment uses Virtual Machines (VMs) as
> MicroCloud cluster members.

Reviewer: ```suggestion
MicroCloud test environment.
```

I don't think this level of detail is needed here. That the guide uses VMs is mentioned in the guide itself, plus that it can be reconfigured by the user for physical machines.

```{toctree}
:maxdepth: 1

Automate a test deployment with Terraform </how-to/terraform_automation>
```

## Engage with us

> Find out how to get community and commercial support, and learn how to make
> contributions to MicroCloud.

Reviewer: ```suggestion
Find out how to get community and commercial support, and learn how to contribute to the MicroCloud project.
```
"make contributions" sounds to me like we're asking for donations 😸
